logging/google/cloud/logging/handlers/_helpers.py

```diff
@@ -36,4 +38,44 @@ def format_stackdriver_json(record, message):
         'severity': record.levelname,
     }
 
+    # Make a best effort to add the trace id.
+    # First try to extract the trace_id from HTTP header, only support flask framework for now.
+    trace_id = parse_flask_request_header()
```

Comment:
I am big time :-1: on putting a **flask-specific** check in a generic method.

A better fix would probably involve allowing a way to register extra checkers, either through a registry in this module or through a registry on a `Client()` object (or a similar type).